src/components/ListItem/ListItemWithImage.js

```diff
@@ -542,11 +543,13 @@ const Addon = (props: AddonProps) => {
       custom,
       customOnRight,
     } = balance;
+    const decimalPlaces = getDecimalPlaces(token);
```

Comment:
I would rename `getDecimalPlaces` to smth like `getTokenFormattingPrecision` since when I read `getDecimalPlaces(token);` I'm thinking about not formatting, but rather the real amount of decimals those the token has (18 for eth, etc)